python/setup.py

```diff
@@ -363,6 +363,22 @@ def build_extensions(self):
         build_ext.build_extensions(self)
 
 
+setup_requires = [
+    'numpy==1.15.4; python_version == "3.4"',
+    'numpy==1.16.4; python_version != "3.4"',
```

Comment:
Can we avoid replicating this code by parsing the `requirements.txt` file?
This should work:
```setup_requires = open('requirements.txt').read().splitlines()```

Reply:
Done